source/includes/example-connect.rst

```diff
@@ -0,0 +1,54 @@
+Gather Connection Information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The source cluster, ``cluster0``, is hosted on the following servers
+and ports:
+
+- clusterOne01.fancyCorp.com: 20020
```

Comment:
[question]

Should there be a space between the full colon and the port numbers?